systemtests/src/main/java/io/enmasse/systemtest/platform/Kubernetes.java

```diff
@@ -173,11 +173,11 @@ public static Kubernetes getInstance() {
             isOpenshift = Exec.execute(Arrays.asList(Kubernetes.getInstance().getCluster().getKubeCmd(), "api-resources"), false).getStdOut().contains("openshift.io");
             ocpVersion = instance.getOcpVersion();
 
-            log.info("Is CRC: {}", isCRC());
-            log.info("Is OpenShift: {}", isOpenShift());
-            log.info("Is OpenShiftCompatible: {}", isOpenShiftCompatible());
+            log.debug("Is CRC: {}", isCRC());
```

Comment:
Because down here it is still `log`

Reply:
yeah it is a different class :) ok I will move back to a log